production.env

```diff
@@ -140,25 +131,27 @@ CERT_VALID_DAYS=365
 OID_VALID_DAYS=365
 GENERIC_VALID_DAYS=365
 OPENSSH_VERSION=7.4
-VGRID_LABEL=VGrid
+VGRID_LABEL=Workgroup
 # Menu options override default and available extra Apps on personal Home page
-#DEFAULT_MENU=
-#USER_MENU=
+DEFAULT_MENU="home files vgrids archives jupyter settings setup logout"
```

Comment:
Perhpas jupyter should be removed or moved to USER_MENU, as it won't work without separate jupyter instances attached?

Reply:
done